index.html

```diff
@@ -3816,6 +3836,56 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
 </pre>
           </aside>
         </section>
+        <!-- Maintain a fragment named "mASt-chunk" to preserve incoming links to it -->
+
+        <section id="mASt-chunk">
+          <h2><span class="chunk">mASt</span> mastering metadata</h2>
+
+          <p>The four decimal values below correspond to the four-byte mASt chunk type field:</p>
+
+          <pre>
+109 65 83 116
+</pre>
+          <p>If present, the <span class="chunk">mASt</span> chunk specifies the mastering display's metadata which includes
+          primaries, white point, maximum luminance, and minimum luminance as specified in [[SMPTE ST 2086]].</p>
```

Comment:
```suggestion
         </p>
```

I do not think it is necessary to repeat the fields that are already specified below.

Reply:
Done